### DISCUSSION TITLE
Visual Studio Crash

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13895] [Copilot Conversations]
I was using it when I got this error in the background tasks panel:

Task Failed - Activating language client Copilot: Failed to initialize Copilot Language Server. Please report at https://github.com/github/feedback

**Comment by AdamCaviness** [comment]
I get the same error as well. The popup give some further detail: "Your task failed with the message: JSON-RPC connection with the remote party was lost before the request could complete."
![Untitled](https://user-images.githubusercontent.com/534790/160879048-3fbc18fb-fb6e-4e62-95e0-a983182f16dd.png)

**Reply by JhonesBR** [reply]
Exact same popup and error for me, if VS is restarted the error appears to be solved, but comeback in a moment

**Comment by Guillote29** [comment]
I have the same error

**Reply by mamfernandes01** [reply]
Me to. I am so curious about this feature. Who can help us?

**Reply by VignRaph** [reply]
Might not be a definitive solution or even the reason for the error, but I had the same message for awhile and simply doing a rebuild of my solution and toggling off then on copilot made it work

**Comment by thekim1** [comment]
Got the same problem
